bin/migrate-oats-data/prod_db_obfuscation/alcs/planning_review/planning_review.py

```diff
@@ -0,0 +1,136 @@
+from common import setup_and_get_logger
+from db import inject_conn_pool
+from ...utils import log_process, get_update_column_query
+from faker import Faker
+
+logger = setup_and_get_logger("prod_data_obfuscation")
+
+
+def obfuscate_pr_related_data():
+    _update_prs()
+    _update_planning_referral()
+    _update_planning_review_decision()
+    _update_planning_review_document()
+
+
+@log_process(logger, "update_with_random_alcs_planning_reviews")
+@inject_conn_pool
+def _update_prs(conn=None):
+    fake = Faker()
+
+    try:
+        with conn.cursor() as cursor:
+            cursor.execute("SELECT uuid FROM alcs.planning_review")
+            record_ids = cursor.fetchall()
+
+            for r_id in record_ids:
+                sql_query = f"""
+                            UPDATE alcs.planning_review 
+                            SET 
+                            {get_update_column_query("document_name")}
+                            WHERE uuid = %s;
+                """
+                cursor.execute(
+                    sql_query,
+                    (
+                        fake.company(),
+                        r_id[0],
+                    ),
+                )
+
+            conn.commit()
+    except Exception as err:
+        conn.rollback()
+        logger.exception(err)
+
+
+@log_process(logger, "update_with_random_alcs_planning_referral")
+@inject_conn_pool
+def _update_planning_referral(conn=None):
+    fake_latin = Faker("la")
+
+    try:
+        with conn.cursor() as cursor:
+            cursor.execute("SELECT uuid FROM alcs.planning_referral")
+            record_ids = cursor.fetchall()
+
+            for r_id in record_ids:
+                sql_query = f"""
+                            UPDATE alcs.planning_referral 
+                            SET {get_update_column_query("referral_description")},
+                            {get_update_column_query("response_description")}
+                            WHERE uuid = %s;
+                """
+                cursor.execute(
+                    sql_query,
+                    (
+                        fake_latin.sentence(nb_words=5),
+                        fake_latin.sentence(nb_words=5),
+                        r_id[0],
+                    ),
+                )
+
+            conn.commit()
+    except Exception as err:
+        conn.rollback()
+        logger.exception(err)
+
+
+@log_process(logger, "update_with_random_alcs_planning_review_decision")
+@inject_conn_pool
+def _update_planning_review_decision(conn=None):
+    fake_latin = Faker("la")
+
+    try:
+        with conn.cursor() as cursor:
+            cursor.execute("SELECT uuid FROM alcs.planning_review_decision")
+            record_ids = cursor.fetchall()
+
+            for r_id in record_ids:
+                sql_query = f"""
+                            UPDATE alcs.planning_review_decision 
+                            SET {get_update_column_query("decision_description")}
```

Comment:
the same as above. In general, if there are nulls in non-obfuscated data, try to keep all nulls as is. Otherwise, it could lead to missing bugs in the finished product with actual data from production